add initial changes

martgil opened the pull request:
This PR will change/update the following:

1. Rephrase passphrase text prompt
2. Added tooltip for passphrase short description.

close #4024  // if this PR closes an issue

issue #0000 // if it doesn't close the issue yet

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)
- Difficult to test (explain why)
- Not worth testing
- Tests will be added later (issue #...)
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

tomholub reviewed (commented):
@martgil please keep it as simple as possible, there is no need for the tooltip

martgil commented:
Hello sir, @rrrooommmaaa. I just removed the tooltip right now and skip all the tests for the tooltip in the previous commit. And I added a new simplified test.
